src/view/components/DataTable/index.tsx

```diff
@@ -81,6 +81,12 @@ export default class DataTable extends React.Component<PropsType, OwnStateType>
       <ListItem style={{ justifyContent: 'center' }} button={true} onPress={this.props.onPressEntry(item.uuid)}>
         <Grid style={{ justifyContent: 'center' }}>
           {this.formatTableValues(Object.values(item))}
+          <Col style={{ width: 10 }}>
+            <Icon
+              name="ios-arrow-forward"

```

Comment:
Do you think we can change this to arrow-up? since the page animation slides from bottom?